src/js/utils/__tests__/rxjsUtils-test.js

```diff
@@ -19,12 +20,25 @@ describe("linearBackoff", function() {
 
       // In test env we don't want to wait for the real wall clock
       // so we encode time intervals with a special helper `m.time`
-      const result = source.retryWhen(linearBackoff(2, m.time("--|")));
+      const result = source.retryWhen(linearBackoff(m.time("--|"), 2));
 
       m.expect(result).toBeObservable(expected);
     })
   );
 
+  it(
+    "retries infinitey when no maxRetries given",
+    marbles(function(m) {
+      m.bind();
+
+      const source = m.cold("1--2#");
+      const expected = m.cold("1--2--1--2----1--2------1--(2|)");
```

Comment:
but that is 3 retries, not 8, no?

Reply:
8 events, not retries 👍

Reply:
Sorry I still don't understand the math behind

Reply:
Oh true! I'm sorry :)